### PR TITLE
Add more channels for slack notification

### PR DESCRIPTION
**Pull request by liangxia**
Follow up of https://github.com/openshift/release/pull/57911

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/.config.prowgen [hunk 1]
```diff
@@ -62,6 +84,60 @@ slack_reporter:
...
+  report_template: '{{if eq .Status.State "success"}} :rainbow: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :rainbow:
+                    {{else}} :volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :volcano: {{end}}'
+
+- channel: '#ocm-qe-prow-ci-jobs'
+  excluded_variants: # It is possible to exclude notifications from specific variants, for example, when you don't require them from older releases
+  - some-old-release
+  job_names: # Listing of job names (ci-operator's 'as' field) that this configuration applies to
+  - aws-rosa-sts-shared-vpc-f28
+  - aws-ocm-osd-ccs-f28
+  - aws-rosa-sts-f28
```
Comment (yasun1) [inline ci-operator/config/openshift/openshift-tests-private/.config.prowgen:134]: The current channel for these three jobs needs to be corrected in Prow. All of them which run the OCP e2e test should be in `managed-hypershift-ci-watcher`. Only the jobs run the OCM testing are reported to `ocm-qe-prow-ci-jobs`. 

Please help update them acordingly. Thanks very much.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/.config.prowgen:122]: Fix in commit [9665321](https://github.com/openshift/release/pull/57921/commits/9665321356b1c251d42f837586306d89e5e618a8)

Comment (yasun1) [reply ci-operator/config/openshift/openshift-tests-private/.config.prowgen:122]: Thanks

**Pull request merged**